File: 2nd_semester/antikimenostrefis_programmatismos/mathima5/unipi/OOP/mathima5/anonymization/Student.java
package unipi.OOP.mathima5.anonymization;

public class Student implements IRead{
    // student fields
    String am;
    String email;

    // 1st constructor
    public Student() {}

    // 2nd constructor
    public Student(String am, String email) {
        this.am = am;
        this.email = email;
    }

    // student method
    void sayHello(){
        System.out.println("Hello from "+am);
    }

    // override build-in method
    @Override
    public String toString() {
        return "Student{ am='"+am+"', email='"+email+"'}";
    }

    // override interface method
    @Override
    public void doRead(String text) {
        System.out.println("I am Student 1 that reads "+text);
    }
}

File: 4th_semester/iatrikh_plhroforikh/mathima3/WebApplication2/WebApplication2/Model/Student.cs
namespace WebApplication2.Model
{
    public class Student
    {
        public String AM { get; }
        public String fullName { get; }
        public String email { get; }
        public Student(string AM, string fullName, string email)
        {
            this.AM = AM;
            this.fullName = fullName;
            this.email = email;
        }
    }
}
